index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    
+    <header class="toolbar">
+     
+       <nav class="logo"><a  href="#"><img src="Images\Logo Placeholder.png" alt="logo" /></a></nav> 
+      <img class="icons" src="Images\Enabled.png" alt="threeline" />
+  </header>
```

Comment:
Great job using the semantic `<header>` tag! 💯 However, there's still room for improvement in this section:

- [ ] Please consider adding the hamburger icon inside your `<nav>` tag, as it represents a key element of your page's navigation! ⛵ 

```suggestion
    <header class="toolbar">
       <nav class="logo">
         <a  href="#"><img src="Images\Logo Placeholder.png" alt="logo" /></a> 
         <img class="icons" src="Images\Enabled.png" alt="threeline" />
       </nav>
  </header>
```

- [ ] Stylewise, please consider removing the space at the top of your toolbar. One of the main goals of this project is that you showcase your ability to follow a given design as closely as possible! 🤓 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/206244188-9c2f423a-ce42-455b-83a4-c14d7f9d26a4.png) | ![image](https://user-images.githubusercontent.com/77069011/206244038-1334f81e-9f57-4cff-89eb-a3da0a823796.png) |